sdk/tests/deqp/functional/gles3/es3fShaderDerivateTests.js

```diff
@@ -647,7 +649,27 @@ goog.scope(function() {
         /** @type {Array<number>} */ var indices = [0, 2, 1, 2, 3, 1];
 
         gl.clearColor(0.125, 0.25, 0.5, 1.0);
-        gl.clear(gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT | gl.STENCIL_BUFFER_BIT);
+        // We can't really call clear() on gl.COLOR_BUFFER_BIT here as in c++ deqp.
+        // The fbo format might be of integer type and WebGL2 requires an INVALID_OPERATION to be generated.
+        var formatObj = gluTextureUtil.mapGLInternalFormat(fboFormat);
+	var fmtClass = tcuTexture.getTextureChannelClass(formatObj.type);
+	switch (fmtClass) {
+            case tcuTexture.TextureChannelClass.FLOATING_POINT:
+            case tcuTexture.TextureChannelClass.SIGNED_FIXED_POINT:
+            case tcuTexture.TextureChannelClass.UNSIGNED_FIXED_POINT:
+                gl.clear(gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT | gl.STENCIL_BUFFER_BIT);
+                break;
+            case tcuTexture.TextureChannelClass.UNSIGNED_INTEGER:
+                gl.clearBufferuiv(gl.COLOR, 0, new Uint32Array(4));
```

Comment:
I don't know how to map to int and uint values.  The semantic meaning is unclear.  Also, I assume the C++ deqp behavior is leave the int and uint buffers untouched when call clear(), so its the same as clearing them to 0.  This is the least behavioral change - may not be the best behavior though.  Do you have suggestions?